src/materials/MeshPhongMaterial.js

```diff
@@ -177,6 +177,23 @@ THREE.MeshPhongMaterial.prototype.toJSON = function () {
 	if ( this.blending !== THREE.NormalBlending ) data.blending = this.blending;
 	if ( this.side !== THREE.FrontSide ) data.side = this.side;
 
+	if(this.map !== null && this.map !== THREE.Texture ) data.map = this.map.uuid;
```

Comment:
I've refactored the whole thing 😀
https://github.com/mrdoob/three.js/commit/25ffb41e6467468d26a9dc317926f9832f17da35